addons/sale_gift_card/tests/test_buy_gift_card.py

```diff
@@ -2,7 +2,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import Command
-from odoo.addons.gift_card.tests.common import TestSaleGiftCardCommon
+from odoo.addons.sales_gift_card.tests.common import TestSaleGiftCardCommon
```

Comment:
wring import => sale_gift_card

you should grep 'sales_gift_card' everywehere and replace it by 'sale_gift_card'